kubernetes/api_deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
         - name: apikey
           mountPath: /etc/kubernetes/apikey
           readOnly: true
+        - name: plugin-login-secrets
+          mountPath: /etc/secrets
```

Comment:
okay. Adding in an environment variable for the 'SECRETS_DIR' that will be able to be change